### PR TITLE
Adding link for already registered users

### PR DESCRIPTION
**Pull request by PAPERPANKS**
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- Added a link to login page under registration form
- Preview Link
  - Post commit view
    screenshot from 2018-10-26 14-30-17
  
  - Before commit view
    screenshot from 2018-10-26 14-26-57

## Description / Changes

- A navigation link to the login page is added for the already registered users.
- A separator line `<hr/>` is also added to distinguish both paragraphs.

### CODE REVIEW

File: page-register.html [hunk 1]
```diff
@@ -674,6 +674,10 @@ <h4 class="uppercase">Register Here</h4>
                                 <p class="mb0">By signing up, you agree to our
                                     <a href="#">Terms Of Use</a>
                                 </p>
+                                <hr style="margin: 8px auto;" />
```
Comment (simsausaurabh) [inline page-register.html]: Do not add style directly to the tags, use css file for adding parameters.

Comment (realslimshanky) [reply page-register.html]: @simsausaurabh totally going with your words I would like to suggest not using inline definitions for the style. Please move them to `css/custom.css`.

Comment (PAPERPANKS) [reply page-register.html]: ok I'll

**Comment by PAPERPANKS** [comment]
@realslimshanky issue updated and resolved in #70

**Pull request closed without merging**